backend/API/swagger_API.yaml

```diff
@@ -49,153 +44,38 @@ paths:
-      responses: 
-        "200":
-          description: "OK"
-          schema:
-            type: "array"
-            items:
-              $ref: "#/definitions/Student"
-        "401":
-          description: "Unauthorized" 
-        "403":
-          description: "Forbidden"
-        "500":
-          description: "Internal Server Error"
-    patch:
-      tags:
-      - "Student"
-      summary: "Add additional feedback"
-      description: "Add additioinal feedback for existing student"
-      operationId: "updateStudent"      
-      consumes:
-      - "application/json"
-      produces:
-      - "integer"
-      parameters: 
-      - in: path
-        name: studentID
-        type: integer
-        required: true
-        description: "student ID number"
-      - in: "body"
-        name: "body"
-        description: "Student Feedback"
-        required: true
-        schema:
-          $ref: "#/definitions/Student"        
-      responses: 
-        "200":
-          description: "OK"      
-        "401":
-          description: "Unauthorized" 
-        "403":
-          description: "Forbidden"
-        "405":
-          description: "Method Not Allowed"
-    delete:
-      tags:
-      - "Student"
-      summary: "Remove Student & Feedback"
-      description: "Delete student, and all comments for student"
-      operationId: "removeStudent"      
-      parameters:
-      - in: path
-        name: studentID
-        type: integer
-        required: true
-        description: "student ID number"
-      responses:
-        "200":
-          description: "OK"        
-        "401":
-          description: "Unauthorized" 
-        "403":
-          description: "Forbidden"
- 
-
-  /Student/{studentID}/AIFeedback/:
-    post:
-      tags:
-      - "AIFeedback"
-      summary: "Invalid operation"
-      description: "Posting comments not supported, use /Student/{studentID} endpoint"
-      consumes: []
-      produces: []
-      parameters:
-      - in: path
-        name: studentID
-        type: integer
-        required: true
-        description: "student ID number"
-      responses: 
-        "401":
-          description: "Unauthorized" 
-        "403":
-          description: "Forbidden"
-        "405":
-          description: "Method Not Allowed"   
-    get:
-      tags:
-      - "AIFeedback"
-      summary: "Returns AI Feedback"
-      description: "Return AI Feedbach Coach assistance based on a new comment for a particular student. Will add comment to student repository, but for adding many comments use /Student/{studentID} endpoint."
-      operationId: "commentFeedback"
-      consumes:
-      - "application/json"
-      produces:
-      - "string"
       parameters:
-      - in: path
-        name: studentID
-        type: integer
-        required: true
-        description: "student ID number"
       - in: "body"
         name: "body"
-        description: "Student Feedback"
+        description: "Student feedback"
```

Comment:
Again we have both Student and AI mentioned. Might be best to choose just one for consistency.

Reply:
Agreed, updated document. Thanks!